Binance.Net/Clients/SpotApi/BinanceClientSpotApiExchangeData.cs

```diff
@@ -142,6 +146,61 @@ public async Task<WebCallResult<BinanceExchangeInfo>> GetExchangeInfoAsync(IEnum
             _log.Write(LogLevel.Information, "Trade rules updated");
             return exchangeInfoResult;
         }
+
+        /// <inheritdoc />
+        public async Task<WebCallResult<BinanceExchangeInfo>> GetExchangeInfoAsync(IEnumerable<string> symbols, bool permissions = false, CancellationToken ct = default)
```

Comment:
https://github.com/HypsyNZ/BinanceAPI.NET/blob/main/API-Test/Tests/ExchangeInfoTest.cs

Reply:
So if I understand correctly this will accept either symbols or permissions as string input? What is the reason to do it this way instead of just with the AccountType[] parameter? 
I think if you just add the AccountType[] parameters to the existing `GetExchangeInfoAsync(IEnumerable<string> symbols, CancellationToken ct = default)` implementation and maybe have another overload with just `AccountType[] permissions` as parameter which points towards the existing implementation it would also work fine right?

Reply:
This method handles both situations as a string

```
string[] symbols = { "BTCUSDT" , "DOGEUSDT" };
string[] permissions = { "SPOT", "MARGIN", "LEVERAGED", "TRD_GRP_002", "TRD_GRP_003", "TRD_GRP_004", "TRD_GRP_005" };
```

You could create an overload I guess I just like to do it this way

Reply:
also `permissions` and `symbols` have the same signature so creating an overload for both of them as a `string` is an error which is why I chose this method. 

![same sig](https://user-images.githubusercontent.com/54571583/193252568-1421b33e-992f-4446-a8e4-68a8602b09e0.png)

Reply:
> What is the reason to do it this way instead of just with the AccountType[] parameter?

I did cover using `AccountType` and `AccountType[]` I just wanted to leave the option to provide it as a `string[]`